Bottom: 9b17e78032e2fb165d1aa213a625e7cace84f45f
Top:    dcbfb2c35318794c22c1c7bc76753c2213d56e01
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 23:10:26 +0100

Refresh of add-a-hid-api-call-notify_save

---

diff --git a/src/hid.h b/src/hid.h
index b1f532f..afb33e7 100644
--- a/src/hid.h
+++ b/src/hid.h
@@ -567,6 +567,16 @@ typedef enum
      * Any remaining rendering will be flushed to the screen.
      */
     void (*finish_debug_draw)  (void);
+
+    /* Notification to the GUI around saving the PCB file.
+     *
+     * Called with a false parameter before the save, called again
+     * with true after the save.
+     *
+     * Allows GUIs which watch for file-changes on disk to ignore
+     * our deliberate changes.
+     */
+    void (*notify_save_pcb) (bool done);
   };
 
 /* Call this as soon as possible from main().  No other HID calls are
